go-controller/pkg/metrics/master.go

```diff
@@ -156,8 +160,15 @@ var metricV6AllocatedHostSubnetCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of v6 host subnets currently allocated",
 })
 
+var metricIPsecEnabled = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "ipsec_enabled",
+	Help:      "Specifies if IPsec is enabled for this cluster(1) or not(0).",
```

Comment:
```suggestion
	Help:      "Specifies whether IPsec is enabled for this cluster(1) or not enabled for this cluster(0)",
```

If space allows, better to make crystal-clear.

In strings for related Metric PRs, I noticed that your strings DID NOT have end punctuation. My change here is only for consistency -- I don't know the particulars for this text type in your product.

Reply:
I updated this PR with all your suggestions. Thank you again for your time.